### ISSUE TITLE
Is there a target network?

### ISSUE THREAD

**Issue by Akella17** [ikostrikov/pytorch-a2c-ppo-acktr#166]
The way I see it, the training loop in main.py is as follows,
1) num_steps steps are taken in the env (times num_processes) and inserted into Rollout.
2) Compute next_value and returns and add them to the Rollout.
3) agent.update(rollouts)
4) rollouts.after_update()

As the optimizer is only called in the 3rd step, the actor_critic network must remain constant during the rest of the time. Then shouldn't the action_log_probs and old_action_log_probs_batch be the same as they are outputs of the same actor_critic network (unless there is a target actor_critic network which I seem to miss out)?

In other words, I don't see why the ratio term (line 61, a2c_ppo_acktr/algo/ppo.py) should be non-zero.

**Comment by Akella17** [comment]
Nevermind, resolved
